region/tourian/main/Blue Hopper Room.json

```diff
@@ -319,6 +319,25 @@
         }
       },
       "devNote": "FIXME: Add requirements to be able to use this runway."
+    },
+    {
+      "link": [2, 2],
+      "name": "Crystal Flash",
+      "requires": [
+        {"resetRoom": {
+          "nodes": [1],
+          "mustStayPut": false
+        }},
+        "canTrickyJump",
+        "h_canCrystalFlash"
+      ],
+      "note": [
+        "Enter the room from the left door, morph, and wait at the left wall.",
```

Comment:
This is "Blue Hopper Room Roll Under Hoppers", and I'd never been able to reliably get to the right side without one of the hoppers chasing me.  Especially the ceiling one because I'm timing the movement against the bottom one.

Another problem is that you can't reach the door at 2 after CFing.

There are some kill strats that would be safe, if you can kill the bottom guy mostly.  But that's not saved as an obstacle.

Reply:
Added a resetRoom requirement to be able to reset the room using the top door after the CF, because you're right that going back left immediately doesn't seem to work, at least not consistently (or without some additional requirements).